main.js

```diff
@@ -0,0 +1,27 @@
+let calculateAge = ((birthdateString) => {
+    // Converting the birthdate string to a Date object.
+    const birthDate = new Date(birthdateString);
+    const today = new Date();
+    let age = today.getFullYear() - birthDate.getFullYear();
+    // Logic to Check if the birthday has been occured this year or not.
+    const monthDifference = today.getMonth() - birthDate.getMonth();
+    if (monthDifference < 0 || (monthDifference === 0 && today.getDate() < birthDate.getDate())) {
+        age--;
+    }
+    return age;
+});
+
+let displayAge = (() => {
```

Comment:
There's no need to wrap you arrow function in a round brackets. You can write it in this way as well.
const displayAge = () => {
    
}